sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -247,6 +251,125 @@ def test_lasso_cv_positive_constraint():
     assert min(clf_constrained.coef_) >= 0
 
 
+@pytest.mark.parametrize(
+    "test_model, args",
+    [(Lasso, {"tol": 1e-16, "alpha": 0.1}), (LassoLars, {"alpha": 0.1}),
+     (RidgeClassifier, {"solver": 'sparse_cg', "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
+     (Ridge, {"solver": 'sparse_cg', 'tol': 1e-12, "alpha": 0.1}),
+     (BayesianRidge, {}), (ARDRegression, {}), (OrthogonalMatchingPursuit, {}),
+     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
+     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
+     (MultiTaskLasso, {"tol": 1e-16, "alpha": 0.1}),
+     (Lars, {}), (LinearRegression, {}), (LassoLarsIC, {})
+     ])
```

Comment:
```suggestion
    "test_model, args",
    [(Lasso, {"tol": 1e-16, "alpha": 0.1}),
     (LassoLars, {"alpha": 0.1}),
     (RidgeClassifier, {"solver": 'sparse_cg', "alpha": 0.1}),
     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
     (Ridge, {"solver": 'sparse_cg', 'tol': 1e-12, "alpha": 0.1}),
     (BayesianRidge, {}),
     (ARDRegression, {}),
     (OrthogonalMatchingPursuit, {}),
     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
     (MultiTaskLasso, {"tol": 1e-16, "alpha": 0.1}),
     (Lars, {}),
     (LinearRegression, {}),
     (LassoLarsIC, {})]
 )
```

Reply:
`test_model` should be called -> `LinearModel` since this is a class
`args` should be called -> `params`